engines/bops_admin/app/helpers/bops_admin/application_helper.rb

```diff
@@ -71,5 +71,16 @@ def nav_items
         {text: "Settings", href: profile_path, active: active_page_key?("settings")}
       ]
     end
+
+    def link_to_document(link_text, document, **args)
```

Comment:
Would it make sense to move this into BopsCore and reference it from there? This'll be the fourth version of this helper, I think.

Reply:
Yeah probably should, I'll do it now as a separate PR